app/assets/stylesheets/govuk_publishing_components/components/_modal-dialogue.scss

```diff
@@ -72,11 +72,17 @@ $govuk-modal-wide-breakpoint: $govuk-page-width + $govuk-modal-margin * 2 + $gov
   touch-action: none;
 }
 
-.app-o-template__body--modal {
+.gem-o-template--modal {
+  @include govuk-media-query($media-type: screen) {
+    overflow-y: inherit;
```

Comment:
I'm not sure I understand this change. Why not just put `overflow: hidden` here? From playing around with it, it seems that you wouldn't then need to set hidden on the body element. Or am I missing something?

Reply:
Indeed the overflow: hidden property could be in theory set to either `<html>` or `<body>`. I personally prefer not to add styles to the html element unless absolutely necessary because it is a special element, that's why I decided in the initial work to set it to `<body>` not knowing the overflow of the `<html>` element will be changed.